### PR TITLE
adding information the find information about sendgrid

### PR DESCRIPTION
**Pull request by luanalves**
## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

This change addresses where to get sendgrid DNS data

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by hguthrie** [comment]
@luanalves The Cloud guide is no longer versioned and no longer located in the `guides` folder. Could you update your PR with the latest devdocs master and make your changes to the Sendgrid topic in the  `devdocs/src/cloud/project/` folder? Thanks!

**Comment by hguthrie** [comment]
Thanks for updating. Adding a label because I am waiting on an internal request before I continue the review.

File: src/cloud/project/sendgrid.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ em.example.com IN CNAME <sendgrid.net>
 s1.example.com IN CNAME <s1.example.sendgrid.net>
 s2.example.com IN CNAME <s2.example.sendgrid.net>
 ```
+The information about your account do you find in https://cloud.magento.com/project/{project-id}/setup/project_details in tab Hosting Info
```
Comment (hguthrie) [inline src/cloud/project/sendgrid.md]:
```suggestion
{: .bs-callout-tip}
You can find Sendgrid details for your account in the Onboarding UI. Use the `https://cloud.magento.com/project/{project-id}/setup/project_details` URL and select the **Project Details** > **Hosting Info** tab.
```

**Comment by dobooth** [comment]
Hi @luanalves Any update with this PR?

**Comment by luanalves** [comment]
Hi @hguthrie and @nuzil this request are accepted.

**Comment by hguthrie** [comment]
@dobooth This one is good to go.

**Pull request merged**